Controller/PlatformUIController.php

```diff
@@ -8,16 +8,27 @@
  */
 namespace EzSystems\PlatformUIBundle\Controller;
 
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentValue;
 use EzSystems\PlatformUIBundle\ApplicationConfig\Provider;
+use EzSystems\PlatformUIBundle\Loader\Loader;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 class PlatformUIController extends Controller
 {
     /** @var Provider */
     private $configAggregator;
 
-    public function __construct(Provider $configAggregator)
+    /** @var  Loader */
```

Comment:
CS as well: please use the FQN in PhpDoc (phpstorm tip: ctrl+shift+alt+c on the class name, then paste).